Enables `inplace` in the quantized relu

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#24374 Enables `inplace` in the quantized relu**

This is a duplicate to bring back #23704 with diff revision D16634539

Differential Revision: [D16818664](https://our.internmc.facebook.com/intern/diff/D16818664)

Commit:
Enables `inplace` in the quantized relu

This is a duplicate to bring back #23704 with diff revision D16634539

Commit:
Update on "Enables `inplace` in the quantized relu"


This is a duplicate to bring back #23704 with diff revision D16634539

Differential Revision: [D16818664](https://our.internmc.facebook.com/intern/diff/D16818664)

jamesr66a commented on test/test_quantized.py:
```diff
@@ -95,15 +98,24 @@ def test_qrelu(self, X):
+
+        for name, op in ops_under_test_inline.items():
+            qY_hat = qX.clone()
+            op(qY_hat)
```
Can you name `op` to `op_` so that someone scanning the code will be immediately aware this is in-place?

jamesr66a reviewed (approved):
looks fine

Commit:
Update on "Enables `inplace` in the quantized relu"


This is a duplicate to bring back #23704 with diff revision D16634539

Differential Revision: [D16818664](https://our.internmc.facebook.com/intern/diff/D16818664)

Commit:
Update on "Enables `inplace` in the quantized relu"


This is a duplicate to bring back #23704 with diff revision D16634539

Differential Revision: [D16818664](https://our.internmc.facebook.com/intern/diff/D16818664)

Commit:
Update on "Enables `inplace` in the quantized relu"


This is a duplicate to bring back #23704 with diff revision D16634539

Differential Revision: [D16818664](https://our.internmc.facebook.com/intern/diff/D16818664)

Commit:
Update on "Enables `inplace` in the quantized relu"


This is a duplicate to bring back #23704 with diff revision D16634539

Differential Revision: [D16818664](https://our.internmc.facebook.com/intern/diff/D16818664)
